### PR TITLE
Feature/add readme

### PR DESCRIPTION
**Pull request by eedwards0**
DACT-177

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,110 @@
...
+#### Using the REST API directly
+The API specification can be found - TBA
+
+##### Officer Filing API
+Add link to the public API documentation when available
+
+## Design
+The following information will only be accessible from within the Companies House network.
+
+* https://companieshouse.atlassian.net/wiki/spaces/DACT/pages/3649699904/Directors+Service
```
Comment (PhilipGlover) [inline README.md:92]: mark up for the link isn't right

File: README.md [hunk 2]
```diff
@@ -1 +1,109 @@
-# officer-filing-api
\ No newline at end of file
+# officer-filing-api
+This API is for
```
Comment (PhilipGlover) [inline README.md:2]: indeed what is it for?

File: README.md [hunk 3]
```diff
@@ -1 +1,109 @@
...
+    ```
+    ./bin/chs-dev modules enable officer-filing
+
+    tilt up
+    ```
+1. To enable development workflow for the service run the command: `./bin/chs-dev development enable officer-filing
+` 
+
+1. To disable development workflow for the service run the command: `./bin/chs-dev development disable officer
+-filing
```
Comment (PhilipGlover) [inline README.md:65]: missing back tick at end

Comment (eedwards0) [reply README.md]: Added

File: README.md (hunk 2 above)
Comment (eedwards0) [reply README.md]: TBA added under title

File: README.md (hunk 1 above)
Comment (eedwards0) [reply README.md]: Added titles for hyperlinks

**Review by PhilipGlover — APPROVED** [review]
All looks good to me

**Pull request merged**